src/Core/Search/Filters/ProductCombinationFilters.php

```diff
@@ -56,6 +56,7 @@ public function __construct(array $filters = [], $filterId = '')
             throw new InvalidArgumentException(sprintf('%s filters expect a product_id filter', static::class));
         }
 
+        $this->isPersisted = false;
```

Comment:
so this line removes the ability to persist the combination filters to db right?

Reply:
Yep exactly, this way the `PersistFiltersBuilder` and the `RepositoryFiltersBuilder` are skipped

I don't think we aim to persist these filters anyway, but to be honest it hides my incompetence to clear the attribute filters when they were empty 😅  I didn't find a way to send empty array, so the last select attribute was always in the DB and I couldn't clear it

This problem will probably rise up again sometime, but in this case it seemed to me that not persist the filters at all was also a suitable solution That makes me think that I could have kept a simple filterId instead of a dynamic one (injecting the product ID was initially to save filters for each product and not globally), but I guess it doesn't hurt now that it's here and it can be used as an example as well